packages/astro/test/fixtures/core-image-base/tsconfig.json

```diff
@@ -1,9 +1,11 @@
 {
-  "extends": "astro/tsconfigs/base",
+  "extends": "astro/tsconfigs/strict",
```

Comment:
When not strict, the content collection fields are nullable, which makes them invalid for images